config/routes.rb

```diff
@@ -1,8 +1,8 @@
 Rails.application.routes.draw do
   root "static_pages#home"
 
-  get "static_pages/home"
-  get "static_pages/help"
-  get "static_pages/about"
-  get "static_pages/contact"
+  get  "/help",    to: "static_pages#help"
```

Comment:
```suggestion
  get "/help", to: "static_pages#help"
```
tương tự với những cái dưới